.gitignore

```diff
@@ -89,3 +89,6 @@ $RECYCLE.BIN/
 
 # Python
 *.pyc
+
+# JetBrains IDEs
+.idea
```

Comment:
I'm not sure if this is supposed to be added to the `.gitignore`, though if we're adding Jetbrains IDE config files to the gitignore then `*.iml` should also be ignored.

Reply:
I think it's acceptable, Agreed that .iml should also be ignored

Reply:
I'll add *.iml, my bad. I use PyCharm to edit the docs.